src/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixer.php

```diff
@@ -159,7 +161,7 @@ private function fixSpacesAroundParentheses(Tokens $tokens)
             }
 
             if ($token->isType(DocLexer::T_NONE)) {
-                if (false !== strpos($token->getContent(), "\n")) {
+                if (false !== strpos($token->getContent(), $lineEnding)) {
```

Comment:
I wrote tests with `setWhitespacesConfig()` calls in the first place so implementing the interface was required. But indeed, defining the whitespaces config isn't actually required at all.